osu.Game.Tests/Rulesets/Scoring/ScoreProcessorTest.cs

```diff
@@ -53,5 +53,237 @@ public void TestSingleOsuHit(ScoringMode scoringMode, HitResult hitResult, int e
 
             Assert.IsTrue(Precision.AlmostEquals(expectedScore, scoreProcessor.TotalScore.Value));
         }
+
+        [TestCase(ScoringMode.Standardised, HitResult.Miss, HitResult.Great, 0)]
+        [TestCase(ScoringMode.Standardised, HitResult.Meh, HitResult.Great, 387_500)]
+        [TestCase(ScoringMode.Standardised, HitResult.Ok, HitResult.Great, 425_000)]
+        [TestCase(ScoringMode.Standardised, HitResult.Good, HitResult.Perfect, 3_350_000 / 7.0)]
```

Comment:
where does the 7.0 come from? should likely be moved out to a `const`

Reply:
Would be nice to have an inline comment (xmldoc on the method?) explaining what these tests are doing, and where the numbers come from. Will be helpful when we inevitably break them and need to update values.

Reply:
7.0 is for the fractional portion. Thoguth it looked easier on the eye, could easily be replaced with double. Also the reason why the expected score is double in this test but int in the test that already existed

Reply:
it's because the test is using perfect as the max judgement, which is 350 - I guess the intent was to avoid specifying inexact values (100/350 does not divide well in base 10). I figured it was tolerable but do agree that it should probably just be a whole number rounded to the nearest unit here, and the comparison should use half precision.

Reply:
it's this one right?

![image](https://user-images.githubusercontent.com/191335/95565905-cc404d80-0a5b-11eb-94f0-9651da3d6ab6.png)

should probably expose that as a public const and consume it here

Reply:
Not exactly, it's the accuracy. Three OK judgements if a PERFECT is the max result gives 300/1400 = 0.2142857... which has no finite decimal expansion.